examples/04_pong/pong.rs

```diff
@@ -75,8 +75,8 @@ fn initialise_paddles(world: &mut World) {
     let mut right_transform = LocalTransform::default();
 
     // Correctly position the paddles.
-    left_transform.translation = [0.0, 0.0, 0.0];
-    right_transform.translation = [ARENA_WIDTH - PADDLE_WIDTH, 0.0, 0.0];
+    left_transform.translation = [0.0, ARENA_HEIGHT / 2.0, 0.0];
```

Comment:
This is not consistent. Left paddle will be halfway outside, while right will be wholly inside.

Reply:
And I know that was there before just felt it was a good opportunity to fix it :)

Reply:
Are you sure? At least it doesn't seem to be the case right now.

Reply:
Look good to me in-game

Reply:
Well look at the x parameter. One is 0, one is window width - paddle size

Reply:
Exactly, which seems right to me.

Reply:
Ah, reference point isn't center point of the paddle?

Reply:
![pong](https://user-images.githubusercontent.com/9071268/31509228-716ab47a-af89-11e7-81b0-65ce24109583.png)

Reply:
How it looks

Reply:
Yeah it's fine, I realized that the paddle isn't centered on its center of mass.

Reply:
No, I don't think so. But I think the y coordinate isn't right yet.

Reply:
Yeah, the origin of the paddle is the upper left corner.

Reply:
Or bottom, not sure

Reply:
You are right, They aren't exactly in the center

Reply:
Should be fixed now, hopefully